numpy/lib/_arraysetops_impl.py

```diff
@@ -433,6 +433,15 @@ def unique_all(x):
     --------
     unique : Find the unique elements of an array.
 
+    Examples
+    --------
+    >>> np.unique_all([1, 1, 2])  # doctest: +NORMALIZE_WHITESPACE
+    UniqueAllResult(
+            values=array([1, 2]),
```

Comment:
There is no whitespace between `(` and `values=` so the formatting must be like this
```suggestion
    UniqueAllResult(values=array([1, 2]),
```

Reply:
This is likely to just fix itself with scipy_doctest BTW. While numpy is on reguide-check, what Matti said.

Reply:
Thank you guys! Doctests passed